### DISCUSSION TITLE
A different way to open the spawner GUI

### DISCUSSION THREAD

**Discussion by JohnnyX071** [OmerBenGera/WildStacker#539] [Implemented Feature Requests] [answered]
Hi, would it be possible to add a toggle in the config that allows the spawner GUI to be opened in another way? For example it would be nice if you could just open it by right clicking on it (if you are not holding other spawners) instead of having to shift + right click

**Comment by OmerBenGera** [comment, marked as answer]
Implemented in latest dev build
